redpanda.yaml

```
@@ -0,0 +1,161 @@
+  version: "25.1.4"
+  description: "Redpanda is a streaming platform based on Apache Kafka API"
+  resources:
+    cpu: 63
+    memory: 140Gi
+  copyright:
+    - license: "Apache-2.0"
+  dependencies:
+    runtime:
+      - bash
+      - e2fsprogs
+      - glibc
+      - krb5-libs
+      - libaio
+      - numactl
+      - openssl
+
+vars:
+  java-version: 21
+
+environment:
+  contents:
+    packages:
+      - abseil-cpp-dev
+      - autoconf
+      - automake
+      - bash
+      - bazelisk-default
+      - binutils
+      - bison
+      - boost-dev
+      - build-base
+      - busybox
+      - c-ares-dev
+      - ca-certificates-bundle
+      - cargo-auditable
+      - cargo-c
+      - ccache
+      - clang
+      - cmake
+      - coreutils
+      - crane
+      - doxygen
+      - fmt-dev
+      - git
+      - glibc-locale-en
+      - gnutls-dev
+      - go
+      - hwloc-dev
+      - isl-dev
+      - krb5-dev
+      - libaio-dev
+      - libcom_err
+      - libsctp-dev
+      - libtool
+      - liburing-dev
+      - libuuid
+      - libxml2-dev
+      - llvm-lld-18
+      - lz4-dev
+      - numactl-dev
+      - openjdk-${{vars.java-version}}-default-jdk
+      - openssl-dev
+      - patch
+      - pcre2-dev
+      - pkgconf-dev
+
+      - ragel
+      - re2-dev
+      - rust
+      - s3cmd
+      - samurai
+      - snappy-dev
+      - util-linux-dev
+      - valgrind-dev
+      - xfs-scrub
+      - xfsprogs-dev
+      - xxhash-dev
+      - yaml-cpp-dev
+      - yamllint
+      - zlib
+      - zstd
+  environment:
+    STABLE_GIT_LATEST_TAG: ${{package.version}}
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: "https://github.com/redpanda-data/redpanda.git"
+      expected-commit: "6cd1d4ef6b8424ee4fd219df94a90abe839a36bc"
+      tag: "v${{package.version}}"
+      destination: redpanda
+
+  - working-directory: redpanda
+    uses: patch
+    with:
+      patches: ../0001-build-update-krb5-configuration-to-include-system-er.patch
+
+  - working-directory: redpanda
+    runs: |
+      mkdir -p ${{targets.destdir}}/usr/bin/
+      mkdir -p ${{targets.destdir}}/usr/lib/redpanda
+      bazel build --verbose_failures --action_env=STABLE_GIT_LATEST_TAG=${{package.version}} --show_loading_progress --config=release --strategy=sandboxed --jobs=16 --local_resources=memory=28000 --local_resources=cpu=22 //...
+      mv bazel-bin/src/v/redpanda/redpanda ${{targets.destdir}}/usr/bin/
+      mv bazel-bin/src/go/rpk/cmd/rpk/rpk_/rpk ${{targets.destdir}}/usr/bin/
+      # Copy redpanda config file to default location /etc/redpanda
+      mkdir -p ${{targets.destdir}}/etc/redpanda
+      cp conf/redpanda.yaml ${{targets.destdir}}/etc/redpanda/redpanda.yaml
+
+  - uses: strip
+
+subpackages:
+  - name: ${{package.name}}-compat
+    pipeline:
+      - runs: |
+          tag=v${{package.version}}
+          crane export redpandadata/redpanda:${tag} - | tar -xvf - entrypoint.sh
+          mv entrypoint.sh ${{targets.subpkgdir}}/
```

```suggestion
          mkdir -p ${{targets.subpkgdir}}/
          mv entrypoint.sh ${{targets.subpkgdir}}/
```

The mkdir command will fix the CI error about being unable to mv entrypoint.sh